src/components/AccountForms/RegistrationForm/RegistrationForm.test.js

```diff
@@ -94,9 +94,12 @@ describe('<RegistrationForm />', () => {
       fireEvent.input(getByTestId('email'), {
         target: { value: 'johndoe' },
       });
-      fireEvent.input(getByTestId('userName'), {
+
+      const userName = getByTestId('userName');
```

Comment:
it seems like this const is only used once? rather in-line it unless i'm missing something

Reply:
good catch! I had some issues here before and i forgot to clean up :+1: